lib/surgex/parser/parsers/id_parser.ex

```diff
@@ -4,18 +4,26 @@ defmodule Surgex.Parser.IdParser do
   alias Surgex.Parser.IntegerParser
 
   @type errors :: :invalid_identifier | IntegerParser.errors()
+  @type option :: {:max, integer()}
 
-  @spec call(term()) :: {:ok, integer | nil} | {:error, errors}
-  def call(nil), do: {:ok, nil}
-  def call(""), do: {:ok, nil}
+  # This parser already validates that ids cannot be below zero
+  # therefore we don't handle the Postgres min integer case
```

Comment:
Maybe please extend the comment with the information that it's a max of the `Serial` type (which ids are by default, but not foreign keys afaik), not integer in this case which maybe at some point we might need to switch from into the `BigSerial` which has the max of `9,223,372,036,854,775,807`

Reply:
What I mean is to include the information where the value comes from.